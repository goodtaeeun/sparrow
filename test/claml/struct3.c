struct __pthread_cond_s {
  __extension__ unsigned long long int __wseq;
  struct {
    unsigned int __low;
    unsigned int __high;
  } __wseq32;
};

int main() {
  struct __pthread_cond_s a;
  return 0;
}
