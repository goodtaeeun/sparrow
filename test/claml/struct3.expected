/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 3 "struct3.c"
struct __anonstruct_0 {
  unsigned int __low;
  unsigned int __high;
};
#line 1 "struct3.c"
struct __pthread_cond_s {
  unsigned long long __wseq;
  struct __anonstruct_0 __wseq32;
};
#line 9 "struct3.c"
int main(void) {
  struct __pthread_cond_s a;

  {
#line 11
    return (0);
  }
}
